quarkus-spring-web codestart can't be used with quarkus-resteasy-reactive

### Describe the bug

quarkus-spring-web codestart can't be used with quarkus-resteasy-reactive

```
Caused by: java.lang.IllegalStateException: Please make sure there is only one provider of the following capabilities:
capability io.quarkus.rest is provided by:
  - io.quarkus:quarkus-resteasy-reactive::jar:2.6.3.Final
  - io.quarkus:quarkus-resteasy::jar:2.6.3.Final
```

quarkus-spring-web codestart Forces RESTEasy Classic - https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/spring-web-codestart/codestart.yml#L19

### Expected behavior

quarkus-spring-web codestart can be used with quarkus-resteasy-reactive

### Actual behavior

quarkus-spring-web codestart can't be used with quarkus-resteasy-reactive

### How to Reproduce?

 - `mvn io.quarkus.platform:quarkus-maven-plugin:2.6.3.Final:create -DprojectGroupId=my-groupId -DprojectArtifactId=app-generated-skeleton -DprojectVersion=1.0.0-SNAPSHOT -DpackageName=org.my.group -DquarkusRegistryClient=false -DplatformArtifactId=quarkus-bom -DplatformVersion=2.6.3.Final -Dextensions=quarkus-spring-web,quarkus-resteasy-reactive`

 - `mvn -f app-generated-skeleton quarkus:dev`

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

Java 17

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.6.3.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/spring-web-codestart/codestart.yml
+++ b/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/spring-web-codestart/codestart.yml
@@ -16,6 +16,6 @@ language:
       package-name: org.acme
     dependencies:
       - io.quarkus:quarkus-spring-web
-      - io.quarkus:quarkus-resteasy-jackson
+      - io.quarkus:quarkus-resteasy-reactive-jackson
     test-dependencies:
       - io.rest-assured:rest-assured

